src/style.css

```diff
@@ -1,3 +1,61 @@
-body {
+body, h1, h2, h3, p, ul {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  background-color: white;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  left: 0;
+  right: 0;
+  align-items: center
+}
+
+.logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__link {
+  position: relative;
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  list-style: none;
+  text-decoration: none;
+  color: #000;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  margin-left: 20px;
+  text-transform: uppercase;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__list .nav__link.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+  background: #00ACDC;
```

Comment:
Move all repeatable colors to the CSS variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

Reply:
there is another problem, why does everything work when npm starts, but when I deploy it, it's like this?
![image](https://github.com/mate-academy/layout_moyo-header/assets/144326045/b11a43a5-0af5-4f92-a851-8856731880c9)
![image](https://github.com/mate-academy/layout_moyo-header/assets/144326045/b2400f3b-c7b1-4638-907b-cb2e07fc1f74)